src/components/UI/Timer/Timer.tsx

```diff
@@ -20,10 +21,22 @@ export const Timer: React.FC<TimerProps> = (props: TimerProps) => {
   }, []);
 
   if ((contactStatus && contactStatus === 'INVALID') || contactBspStatus === 'NONE' || !time) {
-    return <ContactOptOutIcon />;
+    return (
+      <Tooltip
+        tooltipClass={`${styles.Tooltip} ${styles.TimerEndTooltip}`}
+        tooltipArrowClass={styles.TooltipArrow}
+        title="User has not opted in to your number. You can only send message them when they initiate the conversation."
+        placement="bottom"
+      >
+        <ContactOptOutIcon />
+      </Tooltip>
+    );
   }
 
   let timerStyle = styles.TimerNormal;
+  let tooltipStyle = styles.TimerNormalTooltip;
+  let tooltip =
+    'Session window is open to message this contact. Learn more about the WhatsApp session window here.';
```

Comment:
```suggestion
   Learn more about the WhatsApp session window here.';
```
Do we need this line since there is no link provided?